content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -124,6 +124,12 @@ With the help of the Big Cloud Fabric's virtual pod multi-tenant architecture, c
 
 BCF was recognized by Gartner as a visionary in the latest [Magic Quadrant](https://go.bigswitch.com/17GatedDocuments-MagicQuadrantforDataCenterNetworking_Reg.html). One of the BCF Kubernetes on-premises deployments (which includes Kubernetes, DC/OS & VMware running on multiple DCs across different geographic regions) is also referenced [here](https://portworx.com/architects-corner-kubernetes-satya-komala-nio/).
 
+### Calico
+
+[Calico](https://docs.projectcalico.org/) is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. Calico supports multiple data planes including: a state-of-the-art pure Linux eBPF dataplane, a standard Linux networking dataplane, and a Windows HNS dataplane. Whether you prefer cutting edge eBPF, or the familiarity of the standard primitives that existing system administrators already know, you’ll get the same, easy to use, base networking, network policy and IP address management capabilities, that have made Calico the most trusted networking and network policy solution for mission-critical cloud-native applications.
+
+Calico supports a broad range of platforms including Kubernetes, OpenShift, Docker EE, OpenStack, and bare metal services. The largest public cloud providers have selected Calico to provide network security for their hosted Kubernetes services (Amazon EKS, Azure AKS, Google GKE, and IBM IKS) running across tens of thousands of clusters.
```

Comment:
This line isn't needed in Kubernetes documentation. Readers who want to learn about Calico in detail are welcome to follow the link.

Reply:
I would argue that cloud provider support of a network plugin is important high-level information to a Kubernetes user trying to decide on a network plugin. I don't think this phrase is out of place on this page given the existing descriptions.

Reply:
Feature docs are not a place for vendor pitches. For some of the context behind the (settled) discussion on the website content policy, see https://github.com/kubernetes/enhancements/blob/master/keps/sig-docs/20191020-doc-policies-for-third-party-content.md

SIG Docs are working through existing content to make it fit the style guide. We haven't fully updated every page yet in fact, there's a szieable backlog of remaining work.
Typically if someone has said in a comment “I represent vendor C, and vendors B and C have their products overrepresented”, I've mentioned that I'd be happy to review a PR that revises the existing page. SIG Docs does not wish to be in a position of judging content on any criteria other than whether it fits in with the published authoring guidance, and whether is is well written as documentation.

I'd prefer more concise text here, because we prefer concise documentation throughout this repo.
To illustrate another point; Amazon EKS is recommending AWSVPC networking mode for Kubernetes, listing Tigera Calico as an [alternative](https://docs.aws.amazon.com/eks/latest/userguide/alternate-cni-plugins.html). They're entitled to, of course, but it is another reason to favor the shorter text: the longer text suggested here will be harder to maintain as accurate, current documentation.

Reply:
Thanks @sftim , I wasn't aware of that kep, I'll take a look.

To clarify, when you say "this line isn't needed", do you mean everything including and after "The largest public..."?

Reply:
I also meant “Calico supports a broad range of platforms including Kubernetes, OpenShift, Docker EE, OpenStack, and bare metal services.” - what's relevant to Kubernetes docs readers is that the CNI plugin works with Kubernetes.

_Linking_ to a page with detail about support and ecosystem compatibility is not only fine, it's better.

Reply:
Thanks @sftim . I've pushed 7f94c0e to take your earlier suggested edit and a new line on cloud provider integration but linking out instead of listing the vendors inline. Please take another look :)